hastexo/tasks.py

```diff
@@ -364,6 +365,20 @@ def check_stack(self, stack, was_resumed):
                     paramiko.ssh_exception.SSHException,
                     paramiko.ssh_exception.NoValidConnectionsError):
                 self.sleep()
+            except Exception as e:
```

Comment:
Either do something with `e`, or make it just `except:` (PEP-8).

Reply:
We don't want a bare `except:` because we really only want subclasses of `Exception`.  So I'll just drop the `as e`.

Reply:
Oh, OK. Makes sense.